spec/index.bs

```diff
@@ -849,6 +859,166 @@ A [=device=] has the following internal slots:
 
 ### Limits ### {#limits}
 
+Each <dfn dfn>limit</dfn> is a numeric limit on the usage of WebGPU on a device.
+Each [=adapter=] has a set of supported limits, and [=devices=] are created
+with specific limits in place. The device limits are enforced regardless of the
+adapter's limits, and default to a strictly-specified [=limit/baseline=].
+
+One limit value may be <dfn dfn for=limit>better</dfn> than another.
```

Comment:
we should tell more about what this means. For example: "a better limit allows an application to do more complex work", or something like that.

Reply:
Done. While writing this I had the idea to rename "better" to "more strict"/"stricter" (and "worse" to "less strict"); WDYT?